incubator/barbarian/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
`helm/charts` is already under Apache `2.0`, so I don't think there is a need to include the license again.